introduction.md

```diff
@@ -0,0 +1 @@
+Hello. I'm R4DIC4L and I'm now trying improving my Git skills.
```

Comment:
Looks good! Could you also please mention your favorite Code School path in your introduction? This could be Ruby, JavaScript, HTML & CSS, iOS or Electives.